lib/BloqadeQMC/test/error.jl

```diff
@@ -0,0 +1,41 @@
+using Test, BloqadeQMC
+using Statistics
+using Random
+using RandomNumbers
+using Measurements
+using BloqadeQMC: bootstrap
+using Bootstrap: BasicSampling
```

Comment:
do we need this in the tests? `Bootstrap` was only needed for confirming the results IIRC

Reply:
you are right, I forgot to remove this